test/parallel/test-buffer-indexof.js

```diff
@@ -505,20 +505,20 @@ assert.strictEqual(buf_bc.lastIndexOf(Buffer.from('你好'), 7), -1);
 
 // Test lastIndexOf on a longer buffer:
 const bufferString = Buffer.from('a man a plan a canal panama');
-assert.strictEqual(15, bufferString.lastIndexOf('canal'));
-assert.strictEqual(21, bufferString.lastIndexOf('panama'));
-assert.strictEqual(0, bufferString.lastIndexOf('a man a plan a canal panama'));
+assert.strictEqual(bufferString.lastIndexOf('canal'), 15);
+assert.strictEqual(bufferString.lastIndexOf('panama'), 21);
+assert.strictEqual(bufferString.lastIndexOf('a man a plan a canal panama'), 0);
 assert.strictEqual(-1, bufferString.lastIndexOf('a man a plan a canal mexico'));
 assert.strictEqual(-1, bufferString
   .lastIndexOf('a man a plan a canal mexico city'));
 assert.strictEqual(-1, bufferString.lastIndexOf(Buffer.from('a'.repeat(1000))));
```

Comment:
just wondering why some of these were left behind? I guess those can also follow the same change?

Reply:
That was task given to me by @BridgeAR which specified the exact line numbers to change. Maybe he can expand upon why he chose them?

Reply:
I was just checking whether there is any reason that is unknown to me. no worries, thanks for clarifying.